docs/user_guide/model_configuration.md

```diff
@@ -930,6 +930,14 @@ allow the queue to be configured so that individual requests are
 rejected or deferred if their time in the queue exceeds a specified
 timeout.
 
+#### Custom Batching
+
+You can set custom batching rules that work _in addition to_ the default behavior of the dynamic batcher. To do so, you would implement three functions in [tritonbackend.h](https://github.com/triton-inference-server/core/blob/main/include/triton/core/tritonbackend.h) and create a shared library. These functions are `TRITONBACKEND_ModelBatchIncludeRequest`, `TRITONBACKEND_ModelBatchInitialize`, and `TRITONBACKEND_ModelBatchFinalize`.
```

Comment:
NIT: I believe we have a max width of 100 chars per line with the exception of links. Please fix.

Reply:
Thanks! I didn't know what the hard rule was, and I don't believe our formatter works on .md files. Good to know. Fixed. :)